docs/team/candyhy.md

```diff
@@ -9,3 +9,61 @@ PocketEstate enables easy organisation of mass clientele property information th
 location and housing type, that may otherwise be difficult to manage.
 
 Given below are my contributions to the project.
+
+**Code contributed**: [RepoSense link](https://nus-cs2103-ay2021s2.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=&tabOpen=true&tabType=authorship&zFR=false&tabAuthor=candyhy&tabRepo=AY2021S2-CS2103T-T13-4%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false)
+
+#### New Features & Enhancements
+
+* **Enhancement**: Graphical User Interface (GUI)
+    * What it does: Simultaneously displays both property and appointment lists side by side
+    * Justification: This feature improves the product significantly because it enables integration of commands on both property and appointment lists, yet enables the user to conduct distinct commands on both lists and view its effects simultaneously without having to call `list property` or `list appointment` prior to each operation
+    * Highlights: Greying out of properties and appointments with expired deadlines in the GUI, a newly designed Light theme colour scheme with a organised, minimalist design that highlights important details.
+    The implementation was challenging due to limited experience and precise nature of fxml.
+
+<div style="page-break-after: always;"></div>
```

Comment:
This line is used for page breaks when printing to pdf. The maximum page length of ppp is 2 pages, so you can use at most one time only

Reply:
oh okay will correct this